corehq/apps/reports/static/reports/ko/export.manager.js

```diff
@@ -176,6 +176,7 @@ var ExportManager = function (o) {
             url: downloadUrl,
             type: 'POST',
             data: data,
+            traditional: true,
```

Comment:
Parameters that were arrays would have `[]` appended to them. I wasn't sure if it was a Django thing or not, I've never had that happen to me in rails.
http://stackoverflow.com/a/3266648/835696